parsl/tests/test_globus/test_implicit_staging_ftp.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import parsl
+from parsl.app.app import App
+from parsl.data_provider.files import File
+from parsl.tests.configs.local_threads_globus import config
```

Comment:
Do we really need the Globus config for non-globus tests? They will be skipped because globus isn't configured, but I think they should be able to run automatically.